lib/checks/aria/valid-focus-order-widget.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "valid-focus-order-widget",
```

Comment:
This check doesn't do anything with focus order. I think this should be renamed to has-widget-role or something like that. The pass / fail suggestions would also make more sense if we just talk about the element having / not having a widget role.

Reply:
Done.